macro/Markdown.py

```diff
@@ -0,0 +1,17 @@
+import pypandoc
+import requests
+import re
+
+GITHUB_PATTERN = re.compile(r'https?://w*\.?github.com/([^/]+)/([^/]+)/blob/([^/]+)/(.*)')
+RAW_GITHUB_TEMPLATE = 'https://raw.githubusercontent.com/%s/%s/%s/%s'
+
+def macro_Markdown(macro, arg1):
+    content = ''
+    if arg1 is not None:
+        m = GITHUB_PATTERN.match(arg1)
+        if m:
+            arg1 = RAW_GITHUB_TEMPLATE % m.groups()
+        r = requests.get(arg1)
```

Comment:
This will block the main webserver thread prior to returning content, we can't afford to wait very long for a misbehaving remote, or more likely missing. 

We should also add some error checking to provide useful feedback.